src/utils.ts

```diff
@@ -0,0 +1,67 @@
+import type { MessageEntity } from "./deps.deno.ts";
+
+import { FormattedString } from "./format.ts";
+
+interface CaptionEntitiesPayload {
+  caption: FormattedString | string | undefined;
+  caption_entities: MessageEntity[];
+}
+
+interface MediaEntitiesPayload {
+  media: CaptionEntitiesPayload | CaptionEntitiesPayload[];
+}
+
+interface TextEntitiesPayload {
+  text: FormattedString | string | undefined;
+  entities: MessageEntity[];
+}
+
+const captionEntitiesMethod = new Set([
+  "editMessageCaption",
+  "sendAnimation",
+  "sendAudio",
+  "sendDocument",
+  "sendPhoto",
+  "sendVideo",
+  "sendVoice",
+]);
+
+const mediaEntitiesMethod = new Set([
+  "editMessageMedia",
+  "sendMediaGroup",
+]);
+
+const textEntitiesMethod = new Set([
+  "editMessageText",
+  "sendMessage",
+]);
+
+// deno-lint-ignore no-explicit-any
+function isCaptionEntitiesPayload(
+  method: any,
+  _payload: any,
+): _payload is CaptionEntitiesPayload {
```

Comment:
would it not work if define `method` as `string` and `_payload` as `unknown`?

Reply:
Yes, thanks!